pymc/distributions/mixture.py

```diff
@@ -817,10 +817,12 @@ def _hurdle_mixture(*, name, nonzero_p, nonzero_dist, dtype, **kwargs):
 
     nonzero_p = pt.as_tensor_variable(nonzero_p)
     weights = pt.stack([1 - nonzero_p, nonzero_p], axis=-1)
-    comp_dists = [
-        DiracDelta.dist(zero),
-        Truncated.dist(nonzero_dist, lower=lower),
-    ]
+
+    if max_n_steps:
+        truncated_dist = Truncated.dist(nonzero_dist, lower=lower, max_n_steps=max_n_steps)
```

Comment:
Perhaps a bit cleaner if you just copy the default value to the signature of the hurdle function?

Reply:
That's fine. Wasn't sure if we wanted to avoid doubling up on magic numbers, but if you're good with it, I am. Just made the change.